blueprints/engine-blueprint/index.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+const path = require('path');
+const MainBlueprint = require('../../../index');
+
+/*
+  Create an "addon blueprint" that simply defers to our
```

Comment:
Could we add a `TODO` marker to this comment block? Just so it is obvious that there is something incomplete with this.